src/main/resources/messages.properties

```diff
@@ -880,6 +881,9 @@ federalState.GERMANY_SACHSEN_ANHALT=Sachen-Anhalt
 federalState.GERMANY_SCHLESWIG_HOLSTEIN=Schleswig-Holstein
 federalState.GERMANY_THUERINGEN=Thüringen
 
+# Federal States Greece
+federalState.GREECE_ATHENS=Athen
```

Comment:
Maybe we just write "Griechenland", because the public holidays are always based on the country and change `GREECE_ATHENS("gr", "ath")` to `GREECE_ATHENS("gr", "gr")`

Reply:
yes, I thought it that would be nicer too